autoscale-core/src/main/java/com/hpe/caf/autoscale/core/ScalerThread.java

```diff
@@ -173,6 +202,16 @@ private void scaleDown(final int amount)
     {
         LOG.info("Triggering scale down of service {} by amount {}", serviceRef, amount);
         scaler.scaleDown(serviceRef, amount);
+        final InstanceInfo info = scaler.getInstanceInfo(serviceRef);
+        governor.recordInstances(serviceRef, info);
+        backoff = true;
+    }
+
+    public void scaleDownNow() throws ScalerException
+    {
+        scaleDown(1);
+        final InstanceInfo info = scaler.getInstanceInfo(serviceRef);
+        governor.recordInstances(serviceRef, info);
```

Comment:
Aren't these steps already taken by the `scaleDown()` function that you're calling?  Why do they need to be done again?